omod/src/main/webapp/resources/scripts/emr.js

```diff
@@ -87,6 +87,22 @@ var emr = (function($) {
             }
         },
 
+        serverValidationErrorMessage: function(response) {
+        	var errorMessage = "";
+        	if (response.data != null && response.data.error != null) {
+        		errorMessage = response.data.error.message;
```

Comment:
I removed that code because you said that might be too much information to reveal to the user. Sometimes it is also considered as information that can be used to hack the system.